python/tvm/relay/op/strategy/generic.py

```diff
@@ -323,8 +331,10 @@ def compute_conv2d_transpose(attrs, inputs, out_dtype):
         out = topi_compute(
             inputs[0], inputs[1], strides, padding, out_dtype)
         output_padding = get_const_tuple(attrs.output_padding)
-        out = topi.nn.pad(out, [0, 0, 0, 0],
-                          [0, 0, output_padding[0], output_padding[1]])
+        if output_padding[0] != 0 or output_padding[1] != 0:
```

Comment:
Which operators was this breaking on previously?

Reply:
This will cause errors when running dcgan on vta. There are two issues here:
1. if output_padding are all 0, no need to do pad
2. if the size of out.shape is not 4, it will cause problems.

Reply:
> This will cause errors when running dcgan on vta. There are two issues here:
> 
> 1. if output_padding are all 0, no need to do pad
> 2. if the size of out.shape is not 4, it will cause problems.

After rebase, this changes are not needed.